### PR TITLE
Small tweaks to help with dev set up

### PR DESCRIPTION
**Pull request by ignatiusm**
Adds:
- Editing encrypted rails credentials in Docker requires a text editor - adds vim to Dockerfile
- additional env var to `dev` and `test` env files to prevent a warning.
- makefile to make it easy to run setps for getting dev environment set up
- add temp files created as part of makefile to `.gitignore`
- Specific docker tag for playwright image (to help get tests passing)

To come:
- Small tweaks to README.md:
    - instructions for using makefile
    - reducing scope of docker volume prune command
    - warning about git clean
    - guidance for developing without a copy of `config/master.key`

Not done:
- Update nodeJS installation in Dockerfile   **<- Will make a separate PR**
- Possibly adding commands to `makefile` to run docker-compose with user's user ID and group ID so that files created within Docker inherit user's permissions. **<- Will maybe make a separate PR**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -21,7 +21,7 @@ RUN : \
        -ie 's/Suites: bookworm bookworm-updates/Suites: bookworm bookworm-updates oldstable/g' \
        /etc/apt/sources.list.d/debian.sources \
   && apt-get update \
-  && apt-get install -y nodejs yarn libssl1.1 \
+  && apt-get install -y nodejs yarn libssl1.1 vim \
```
Comment (DoxasticFox) [inline Dockerfile:24]: >vim

An excellent choice

File: README.md [hunk 2]
```diff
@@ -112,29 +120,43 @@ email: testuser@email.com
...
 
-##### It is recommended to have Node v14.16.1 (you can check the node version by doing `node -v` from your console). Also do a `yarn -v`  to check if you have yarn installed. If it isn't installed then do `npm install --global yarn`.
+##### It is recommended to have Node v14.16.1 (you can check the node version by doing `node -v` from your console). Also do a `yarn -v` to check if you have yarn installed. If it isn't installed then do `npm install --global yarn`.
 
 ##### Install gems via bundler
 
 ```shell
 bundle install
-```
+````
```
Comment (DoxasticFox) [inline README.md]: There's backticks in this markdown when it's rendered: https://github.com/Australian-Genomics/CTRL/blob/c4050e80961da89fce2f8dc6d13d1f278418e7ad/README.md#other-make-commands

**Review by DoxasticFox — APPROVED** [review]
:ok_hand:

File: README.md (hunk 2 above)
Comment (ignatiusm) [reply README.md]: Whoops - good spotting. I'll fix that.

Comment (ignatiusm) [reply README.md]: Addressed in [54b4a18](https://github.com/Australian-Genomics/CTRL/pull/180/commits/54b4a18281c4e27267b0eed9e0c6584e46ab4a43)

**Comment by DoxasticFox** [comment]
I'm merging this because I need some of these changes 😄

**Pull request merged**